.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
We usually run this on `- '**'` so that stacked changes (where the second PR is targeting the first PR's branch to make reviewing the diff easy) still get CI runs.